sdks/java/managed/build.gradle

```diff
@@ -28,6 +28,9 @@ ext.summary = """Library that provides managed IOs."""
 
 dependencies {
     implementation project(path: ":sdks:java:core", configuration: "shadow")
+    implementation project(":runners:google-cloud-dataflow-java")
+    implementation project(":sdks:java:io:google-cloud-platform")
+    implementation project(":sdks:java:extensions:google-cloud-platform-core")
```

Comment:
I needed these to access `DataflowPipelineOptions::getDataflowServiceOptions()`, but I'd like to avoid this if possible